pkg/flavors/postgres/postgres.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"fmt"
+
+	valid "github.com/asaskevich/govalidator"
+	"github.com/tetratelabs/getenvoy/pkg/flavors"
+)
+
+// Define template parameter names
+const endpoint string = "endpoint"
+const inport string = "inport"
```

Comment:
I think, `endpoint` and `inport` are very obscure argument names (that a user must provide on CLI).

E.g., we could prefix them to make their context more clear.

Reply:
The commands should be added to getnevoy template. Captured it here: https://github.com/tetratelabs/getenvoy/issues/97. The command should list parameters required for a template.